crawler_mcp/crawlers/optimized/cli/pr.py

```
@@ -0,0 +1,475 @@
+        )
+
+
+def _build_filters_from_args(ns: argparse.Namespace) -> dict[str, Any]:
+    f: dict[str, Any] = {}
+
+    # Parse enhanced filters from JSON if provided
+    if hasattr(ns, 'filters') and ns.filters:
+        try:
+            f.update(json.loads(ns.filters))
+        except json.JSONDecodeError:
+            pass  # Ignore invalid JSON
```

Ruff (0.12.2)

Use `contextlib.suppress(json.JSONDecodeError)` instead of `try`-`except`-`pass`

(SIM105)